src/main/java/com/example/training/redis/UserLimit.java

```diff
@@ -0,0 +1,17 @@
+package com.example.training.redis;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+
+@AllArgsConstructor
+@Getter
+@ToString
+public class UserLimit implements Serializable {
+    private final int countMethodGet;
```

Comment:
don't need two counters. If we will add a new method tomorrow? you will add the new field? It is not flexible